app/public/js/system/settings.js

```diff
@@ -9,4 +9,17 @@ window.settings = {};
 window.settings.appVersion = '0.6.4';
 
 // GA >> DO NOT CHANGE OR USE THIS CODE <<
-window.settings.visitor = ua('UA-67310953-1');
\ No newline at end of file
+window.settings.visitor = ua('UA-67310953-1');
+
+// Simple function to check whether OS supports minimize to tray option
+window.settings.traySupport = function () {
```

Comment:
Yes, that's why i added that to `settings.js` instead of `guiConfig.js` because it's used both in `guiConfig` as well in settings view.